sonic-xcvrd/tests/test_xcvrd.py

```diff
@@ -341,8 +341,8 @@ def test_handle_port_config_change(self, mock_select, mock_sub_table):
     @patch('swsscommon.swsscommon.Table')
     def test_get_port_mapping(self, mock_swsscommon_table):
         mock_table = MagicMock()
-        mock_table.getKeys = MagicMock(return_value=['Ethernet0', 'Ethernet4'])
-        mock_table.get = MagicMock(side_effect=[(True, (('index', 1), )), (True, (('index', 2), ))])
+        mock_table.getKeys = MagicMock(return_value=['Ethernet0', 'Ethernet4', 'Ethernet-IB0'])
+        mock_table.get = MagicMock(side_effect=[(True, (('index', 1), )), (True, (('index', 2), )), (True, (('index', 3), ))])
```

Comment:
Thanks for adding this check, can you add the assert fail check too for this.

Reply:
Thanks Judy,  I have added assert check as your suggestion.